### PR TITLE
Support for Aqara Smartplug + Water Sensor

### PR DESCRIPTION
**Pull request by mrsteakhouse**
This adds specific mappings for the Xiaomi Smartplug and Water sensor.
I included parts of the package script to quickly calculate and package small changes in JS files without building the whole project. On my local machine building the project took several minutes even for one changed line of code.

Tested this with my local setup. Even though the smartplug now has only its supported properties, the instantaneous power report seems a bit odd.
#247: The water sensor works now, however i had to re-pair it, until pressing the button made the LED flash quickly. At some point i tested the sensor with [deConz](https://phoscon.de/en/conbee2/software#deconz), which might made it work.

### CODE REVIEW

File: zb-xiaomi.js [hunk 1]
```diff
@@ -476,6 +477,129 @@ const MODEL_IDS = {
       },
     },
   },
+  'lumi.plug.maeu01': {
+    name: 'smartplug',
+    '@type': [
+      'SmartPlug',
+      'EnergyMonitor',
+      'OnOffSwitch',
+      'MultiLevelSensor',
```
Comment (mrstegeman) [inline zb-xiaomi.js:486]: This is not a `MultiLevelSensor`

File: zb-xiaomi.js [hunk 2]
```diff
@@ -476,6 +477,129 @@ const MODEL_IDS = {
...
+        },
+        profileId: PROFILE_ID.ZHA,
+        endpoint: 1,
+        clusterId: CLUSTER_ID.HAELECTRICAL,
+        attr: 'activePower',
+        parseValueFromAttr: 'parseNumericTenthsAttr',
+      },
+      counter: {
+        descr: {
+          '@type': 'InstantaneousPowerProperty',
```
Comment (mrstegeman) [inline zb-xiaomi.js:529]: This property should not have the `InstantaneousPowerProperty` `@type`, as it's not the instantaneous power.

**Review by mrstegeman — CHANGES_REQUESTED** [review]

File: zb-xiaomi.js (hunk 2 above)
Comment (mrsteakhouse) [reply zb-xiaomi.js:529]: Do you have a suggestion which one will be better suited?

Comment (mrstegeman) [reply zb-xiaomi.js:529]: No, you can just omit that field.

File: zb-xiaomi.js (hunk 1 above)
Comment (mrsteakhouse) [reply zb-xiaomi.js:486]: removed

File: zb-xiaomi.js (hunk 2 above)
Comment (mrsteakhouse) [reply zb-xiaomi.js:529]: Alright.

**Review by mrstegeman — APPROVED** [review]

**Pull request merged**